CHANGELOG.next.asciidoc

```diff
@@ -54,6 +54,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Not hiding error in case of http failure using elastic fetcher {pull}11604[11604]
 - Relax validation of the X-Pack license UID value. {issue}11640[11640]
 - Fix a parsing error with the X-Pack license check on 32-bit system. {issue}11650[11650]
+- [libbeat] Escape BOM on JsonReader before trying to decode line {pull}11661[11661]
```

Comment:
Could you please remove the prefix "[libbeat]" from the changelog? It's already placed under the section "Affecting all Beats", so it does not provide any additional information.